plugins/woocommerce-admin/client/settings/settings-slots.js

```diff
@@ -18,6 +18,10 @@ export const possiblyRenderSettingsSlots = () => {
 		},
 		{ id: 'wc_tax_settings_slotfill', scope: 'woocommerce-tax-settings' },
 		{ id: 'wc_settings_slotfill', scope: 'woocommerce-settings' },
+		{
+			id: 'wc_settings_general_site_visibility_slotfill',
+			scope: 'woocommerce-site-visibility-settings',
+		},
```

Comment:
Nice to see this getting used!

Reply:
Nice! useLaunchYourStore looks fantastic! I'll update this PR to use it 👍

Reply:
@psealock Could you also return comingSoon, storePagesOnly, privateLink, and shareKey from the function?